src/Components/auto-page/Leave.tsx

```diff
@@ -11,37 +11,36 @@ function Leave() {
 	const dispatch = useAppDispatch();
 	const leaveValue = useAppSelector(state => state.auto.mobility);
 
-	const [no, setNo] = useState<'outlined' | 'contained'>('outlined');
-	const [yes, setYes] = useState<'outlined' | 'contained'>('outlined');
+	const getButtonStyle = (status) => {
+		if (leaveValue === leaveValues[status]) {
+				return 'contained';
```

Comment:
This appears indented one too many times

Reply:
Fixed with [49f37ff](https://github.com/Team2338/GearScout2024/pull/4/commits/49f37ffe19aabe3d561ecc34df59aa6e7abdfd30)